modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java

```diff
@@ -430,7 +431,11 @@ public static String requestAccessToken(String consumerKey, String consumerSecre
         //Get access token from the response
         JSONParser parser = new JSONParser();
         JSONObject json = (JSONObject) parser.parse(responseString);
-        return json.get("access_token").toString();
+        String accessToken = json.get("access_token").toString();
```

Comment:
Object accessToken = json.get("access_token");
		if (accessToken == null) {
			throw new InvalidCredentialsException("Error occurred while requesting access token.");
		}
		return accessToken.toString();
I modified to throw an exception to prevent returning an empty string as the access token